service/src/test/java/uk/nhs/adaptors/gp2gp/utils/XmlParsingUtility.java

```diff
@@ -0,0 +1,48 @@
+package uk.nhs.adaptors.gp2gp.utils;
+
+import org.w3c.dom.Document;
+
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+
+import java.io.IOException;
+import java.io.StringReader;
+
+public final class XmlParsingUtility {
```

Comment:
I'd suggest to use @NoArgsConstructor instead of 
private XmlParsingUtility() { }

Reply:
Hey @ole4ryb - I believe that `@NoArgsConstructor` would add a `public` no argument constructor, while by providing a private no argument constructor is preferred to prevent unwanted instantiation. Furthermore - I'm not sure if brining in a Lombok annotation for this would worth it in my opinion!  😄 What do you think!

Reply:
I think you can do smth like this:
@NoArgsConstructor(access = AccessLevel.PRIVATE)

Reply:
Now we've reached a fun conclusion! 😄

`private XmlParsingUtility() { }` is 31 characters long,
`@NoArgsConstructor(access = AccessLevel.PRIVATE)` is 48 characters long!